beets/importer.py

```diff
@@ -547,9 +547,22 @@ def _emit_imported(self, lib):
         plugins.send('album_imported', lib=lib, album=self.album)
 
     def emit_created(self, session):
-        """Send the `import_task_created` event for this task.
+        """Send the `import_task_created` event for this task and return a list
+        of tasks which might be returned by plugins.
```

Comment:
Right, your proposed docstring is much better. I'll take that.
Maybe we should call it `emit_created_and_collect_new_tasks_returned_by_plugins`? Just kidding :wink:!
What about `handle_created`?